partner_contact_personal_information_page/README.rst

```diff
@@ -42,6 +65,7 @@ Contributors
 * Jairo Llopis <j.llopis@grupoesoc.es>
 * Matjaž Mozetič <m.mozetic@matmoz.si>
 * Rudolf Schnapka <schnapkar@golive-saar.de>
+* Richard deMeester <richard@willowit.com.au>
 
 Maintainer
```

Comment:
Now you are really doing my head in - I can't see what part of maintainer is wrong?